src/services/infra/InfraService.ts

```diff
@@ -614,10 +615,8 @@ export default class InfraService {
       emailDetails = failureEmailDetails
     }
 
-    await mailer.sendMail(
-      message.requestorEmail,
-      emailDetails.subject,
-      emailDetails.body
-    )
+    const targetEmail = isSuccess ? message.requestorEmail : ISOMER_ADMIN_EMAIL
+
+    await mailer.sendMail(targetEmail, emailDetails.subject, emailDetails.body)
```

Comment:
Should we still send out an email that the process failed with a generic message though? Otherwise the requestor might be stuck waiting for a response when they should actually be trying again

Reply:
@alexanderleegs 
There is no functionality to try again in the first cut of site launch feature! I think messages are useful if we provide actionable steps for them! (actually will check in with nat on this as well) cc. @NatMaeTan

Reply:
won't this imply that our users don't receive a failure email? under our old behaviour, users receive all emails (success + failure) but now they only receive success. 

if the concern is just about information clarity, maybe we can just send them a templated email that their site launch failed and we are aware of the issue + pls contact our helpdesk email

Reply:
Hey @seaerchin this is a fair point!
The thing is here is that we dont differentiate a failed launch and a timeout launch, both are considered failures. My concern here is that there might be false positives that is being sent. 

Since this needs to be out by next week's release for next week's trial, and the next trial after this is not for another month, am opting for this flow for now and to be iterated on.

Reply:
i'm fine with this if

1. it isn't a regression in behaviour (or this has been communicated to end user) -> given that prev site launches were manual and now is automatic, manual -> automatic users might be confused wrt why they aren't receiving any emails
2. @shazlithebestie signs off on this **and** emails the users on actual failure to inform them.

Reply:
> automatic users might be confused wrt why they aren't receiving any emails
for now, there is only one user that will go through this 
> @shazlithebestie signs off on this and emails the users on actual failure to inform them.
nope, I am taking point on this, im handling the site launch for the user that we are trialing this feature out with! Ops has no implications

Reply:
Sgtm

Reply:
from clarification offline, the manual site launch process is done through a form. the email that is inputted in that form is prod ops emails so end user never sees it.